test/integration/helpers/Apps.cs

```diff
@@ -63,5 +63,12 @@ public static string Get(string appKey)
             Init();
             return _testApps[appKey];
         }
+
+        public static string iosTestApp = "iosTestApp";
```

Comment:
shouldn't they be constants?

Reply:
Can't be static and const. I'm referencing these properties in test classes instead of raw string values